QUARKUS-6258.md

```diff
@@ -15,9 +15,16 @@ The list of prioritized features (taken from the issue):
 - Advanced RAG
 - Guardrails 	
 - WebSocket chat integration 	
```

Comment:
"WebSocket chat integration", this refers to the WS Next; what is actual scope? you will have one test? Or test for every relevant WS Next scenario (WS Client, WS Server, broadcasting, pings, uni/multi/non-reactive formats), use it to test every tool/service/model. The reason I am asking, how do you know what is relevant if you don't plan it, state it in this plan and give someone change to challenge that.